### PR TITLE
More internationalization...

### PR DESCRIPTION
**Pull request by HarvelsX**
in future!

Now let's try to move things to using i18next.

### CODE REVIEW

**Comment by Kozjar** [comment]
Great PR!

File: pnpm-lock.yaml
Comment (Kozjar) [inline pnpm-lock.yaml:1]: Can you remove this file? I'm using `yarn` for this project (will add it to the README) maybe I will migrate to the pnpm in the future, but it is better to it in another PR, so I could test it properly.

File: src/pages/auction/PurchaseComponent/PurchaseComponent.tsx [hunk 1]
```diff
@@ -82,13 +82,14 @@ const PurchaseComponent: React.FC<PurchaseComponentProps> = ({
     }
 
     const { slots } = store.getState().slots;
-    const slotNames = slots.map(({ name }) => name || '');
-    const {
-      bestMatch: { rating },
-      bestMatchIndex,
-    } = findBestMatch(message, slotNames);
+    const { search } = new Fuse(slots, {
```
Comment (Kozjar) [inline src/pages/auction/PurchaseComponent/PurchaseComponent.tsx:85]: Why did you choose `fuse.js` over the `string-similarity`? Does it give any benefits?

Comment (HarvelsX) [reply src/pages/auction/PurchaseComponent/PurchaseComponent.tsx:85]: @Kozjar, `string-similarity` is deprecated and not supported.
The library for complex search is good for solving the problem of finding a similar string.
It also allows you to isolate yourself from a particular implementation of the search in favor of using the optimal algorithm chosen by the library.

**Pull request merged**